examples/fluids/problems/newtonian.c

```diff
@@ -202,6 +204,11 @@ PetscErrorCode NS_NEWTONIAN_IG(ProblemData *problem, DM dm, void *ctx, SimpleBC
   if (state_var == STATEVAR_PRIMITIVE && !implicit) {
     SETERRQ(comm, PETSC_ERR_ARG_NULL, "RHSFunction is not provided for primitive variables (use -state_var primitive only with -implicit)\n");
   }
+
+  PetscCall(PetscOptionsScalar("-idl_decay_time", "Effectively inverse of strength of IDL. The timestep is good starting point", NULL, idl_decay_time,
+                               &idl_decay_time, &idl_enable));
```

Comment:
We should explain that `0` is treated like $\infty$. Or perhaps we should use `-1` to mean this is off.

Reply:
I think `-1` to mean off (or really any negative number) makes sense. I think we should treat `0` as an invalid input, right?

Reply:
Sure